test/unit_tests/organisms/Footer-spec.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+var BASE_JS_PATH = '../../../cfgov/unprocessed/js/';
+
+var chai = require( 'chai' );
+var expect = chai.expect;
+//var Footer = require( BASE_JS_PATH + 'organisms/Footer' );
```

Comment:
This isn't commented out in the other specs, but this is fine as it can be uncommented at implementation time.